customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/api/TerminationController.java

```diff
@@ -37,6 +37,16 @@ TerminationDTO create(@RequestBody CreateTerminationDTO request) {
     }
   }
 
+  @PostMapping("/{terminationId}")
+  TerminationDTO update(@PathVariable UUID terminationId,
```

Comment:
With this solution we allow the caller to provide arguments that are not used but can be provided. This make the API unsafe as we can not trust the service to updated all fields we provide. Another solution would be to use a updateTerminationDTO what only contains the 3 fields that will be used and give us the option to add optional fields in the future that will not affect the API.

Reply:
OK! I do get your point and I agree. Will make the change and communicate with @mikeyOne to make necessary changes in Intygsadmin.

Reply:
Fixed!